chart/templates/psp.yaml

```diff
@@ -14,19 +14,34 @@ spec:
   {{- if not .NoInitContainer }}
   allowedCapabilities:
   - NET_ADMIN
+  - NET_RAW
   {{- end}}
+  requiredDropCapabilities:
+  - ALL
+  hostNetwork: false
+  hostIPC: false
+  hostPID: false
   seLinux:
     rule: RunAsAny
-  supplementalGroups:
-    rule: RunAsAny
   runAsUser:
     rule: RunAsAny
+  supplementalGroups:
+    rule: MustRunAs
+    ranges:
+    - min: 1
+      max: 65535
   fsGroup:
-    rule: RunAsAny
+    rule: MustRunAs
+    ranges:
+    - min: 1
+      max: 65535
   volumes:
   - configMap
   - emptyDir
   - secret
+  - projected
+  - downwardAPI
+  - persistentVolumeClaim
```

Comment:
Any particular reason these three types of volumes were added? I couldn't find where in the CP they're required.

Reply:
Just going by the list in https://github.com/linkerd/website/issues/94. They are part of the [recommended minimum set](https://kubernetes.io/docs/concepts/policy/pod-security-policy/#volumes-and-file-systems).